UIP-0027.md

```diff
@@ -0,0 +1,88 @@
+# UIP-27: Transaction service fields
+
+```
+Author:   Dima Saveliev <dima@thirdhash.com>
+Status:   Draft
+Created:  2019-05-07
+```
+
+## Abstract
+
+  This document describes protocol changes in the transaction structure:
+the `nVersion` field is going to be replaced by four separate service fields.
```

Comment:
Maybe name the new fields already here?

Reply:
I skipped the details here in favour of `specification` section, but yes, the list of new fields will be useful here, will do.

Reply:
Done